cime/src/components/data_comps/docn/mct/docn_comp_mod.F90

```diff
@@ -760,7 +760,7 @@ subroutine prescribed_sst(xc, yc, lsize, sst_option, sst)
 
     ! Control
 
-    if (sst_option < 1 .or. sst_option > 10) then
+    if (sst_option < 1 .or. sst_option > 11) then
        call shr_sys_abort ('prescribed_sst: ERROR: sst_option must be between 1 and 10')
```

Comment:
If we do end up using "11" to be for constant SST we should change the error message here, although I don't think we should do that.

Reply:
Yes that error message should be modified.